codespell_lib/data/dictionary.txt

```diff
@@ -24950,6 +24963,8 @@ permmissions->permissions
 permormance->performance
 permssion->permission
 permssions->permissions
+permuatation->permutation
+permuatations->permutations
```

Comment:
Add `permuatate` (with `permuatated`, `permuatates`, `permuatating`)?

Reply:
Indeed. Then perhaps you should forget this `permuatate` suggestion.

As for `permutate->permute`, I would recommend against it because `permutate` is correct, it is found in the dictionary.

Reply:
On the other hand, there are a few hots out there:
https://github.com/search?q=permuatate&type=code

Reply:
Done. I also added `permuate->permute, permutate,` `permution->permutation`.